apps/meteor/app/lib/server/functions/closeLivechatRoom.ts

```diff
@@ -34,9 +35,12 @@ export const closeLivechatRoom = async (
 	}
 
 	if (!room.open) {
-		const subscriptionsLeft = await Subscriptions.countByRoomId(roomId);
-		if (subscriptionsLeft) {
-			await Subscriptions.removeByRoomId(roomId);
+		const { deletedCount } = await Subscriptions.removeByRoomId(roomId, {
```

Comment:
The code removed here was part of a fix some time ago, there's a path of code that will do the removal on happy scenarios. This is for the scenario of a room being closed while still having subs, that's why it counted before removing, to avoid doing the delete query. 

(just making sure the same behavior is kept after this change)

Reply:
I removed the `count` since it is not really required .. if we want to remove them, we do so.. I kept the same behavior regarding returning an error or not by verifying if something was deleted or not (same as count > 0)

Reply:
Thanks man!